### DISCUSSION TITLE
Moving from subdomain to domain

### DISCUSSION THREAD

**Discussion by ljread** [code4recovery/12-step-meeting-list#1076] [General]
Due to host provider increasing the cost of wildcard SSL 3x, we need to move our Wordpress site from the subdomain wp.nwpi.net to the domain. This seems straight forward but wanted to ask if there will be any issues with TSML.

Thanks for the great work by code4recovery.

John

**Comment by joshreisner** [comment]
great! you should be fine, but here are some suggestions anyway:

* go to settings > permalinks and click 'save changes'
* go to a single meeting and click 'update'
* let the meeting guide app support know about the change

**Reply by joshreisner** [reply]
(these are for after you move)
